### PR TITLE
update kubernetes app to utilize cadvisor snap plugin

### PR DESCRIPTION
**Pull request by jtlisi**

### CODE REVIEW

**Comment by woodsaj** [comment]
lets just remove the "/intel/docker/*/*/*/stats/connection/tcp/" for now, and remove the related panels from the dashboards.  We dont want to keep running the docker collector as it does not perform well.  With the way the collector is written, even if we wanted it to just fetch the connection stats, the collector would still try and fetch all stats and then just return the connection stats.

**Comment by jtlisi** [comment]
Gotcha, I'll update that now

**Comment by jtlisi** [comment]
This should be good to go

File: src/components/clusters/clusterConfig.js [hunk 1]
```diff
@@ -426,6 +422,18 @@ var snapTask = {
         "/intel/procfs/meminfo/mem_total_perc": {},
         "/intel/procfs/meminfo/mem_used": {},
         "/intel/procfs/meminfo/mem_used_perc": {},
+        "/intel/procfs/filesystem/*/inodes_free": {},
```
Comment (woodsaj) [inline src/components/clusters/clusterConfig.js:425]: the filesystem metrics need snap-plugin-collector-df, but i dont see it included in the image.  https://github.com/grafana/snap_k8s/pull/4/files

**Review by woodsaj — APPROVED** [review]
Looks greate @jtlisi   just make sure you include the snap-collector-plugin-df plugin in the image.

File: src/components/clusters/clusterConfig.js (hunk 1 above)
Comment (jtlisi) [reply src/components/clusters/clusterConfig.js:425]: Sorry about that I had pushed snap_k8s with the df plugin but I didn't push the commit yet

**Pull request merged**